src/test/java/build/buildfarm/cas/cfc/CASFileCacheTest.java

```diff
@@ -1185,6 +1187,7 @@ public void newInputThrowsNoSuchFileExceptionWithoutDelegate() throws Exception
             /* accessRecorder= */ directExecutor(),
             storage,
             /* directoriesIndexDbName= */ ":memory:",
+            /* zstdBufferPool= */ null,
```

Comment:
Do we have/need a test that actually tests the buffer pool?

Reply:
We need some tests for zstd itself. This is just context for the stream that the CFC creates - tests on input stream contract and tests of zstd content should be good enough.